docs/source/cli/graph-construction/distributed/gsprocessing/distributed-processing-setup.rst

```diff
@@ -45,22 +44,6 @@ you'll need to have the Docker engine installed.
 To install Docker follow the instructions at the
 `official site <https://docs.docker.com/engine/install/>`_.
 
-Install Poetry
```

Comment:
Do we still want to mention anywhere about poetry installation or we just remove all of the existence?

Reply:
We only keep it in the developer guide.